user_docs/en/userGuide.md

```diff
@@ -2243,6 +2243,41 @@ Disabling this option may improve readability.
 
 To toggle show selection from anywhere, please assign a custom gesture using the [Input Gestures dialog](#InputGestures).
 
+##### Formatting display {#BrailleFormattingDisplay}
```

Comment:
This paragraph, as well as the previous one (Show selection), are not at the same position as the controls in the GUI panel. Could you move them so that the doc order match the order of the controls in the GUI?
Thanks.

Reply:
I've moved "Formatting display". I will open a separate PR to reorder the others so that this PR remains focused on a single issue.

Reply:
Created https://github.com/nvaccess/nvda/pull/16898 to address the rest of the section.